src/PluginUtils_TEST.cc

```diff
@@ -30,7 +30,7 @@ TEST(PluginUtils, NormalizeName)
   EXPECT_EQ("::ignition", common::NormalizeName("ignition"));
   EXPECT_EQ("::ignition", common::NormalizeName("::ignition"));
 
-  EXPECT_EQ("::ignition::math", common::NormalizeName("ignition::math"));
+  EXPECT_EQ("::ignition::math", common::NormalizeName("math"));
```

Comment:
I think this should stay as it is

Reply:
I've edited the script now to exclude any namespace cleanups occurring with a leading `"`, so this has been reverted!

I'm not sure what's up with the compilation error, I'll get to it if CI explodes again